tests/test_room_api.py

```diff
@@ -217,13 +240,13 @@ def test_valid_subscription(self, room_api: RoomApi):
                 subscribe_mode=ComponentOptionsHLSSubscribeMode("manual")
             ),
         )
-        assert room_api.hls_subscribe(room.id, ["track-id"]) is None
+        assert room_api.hls_subscribe(room.id, ["peer-id"]) is None
 
     def test_invalid_subscription(self, room_api: RoomApi):
         _, room = room_api.create_room(video_codec=CODEC_H264)
         _ = room_api.add_component(room.id, options=HLS_OPTIONS)
         with pytest.raises(BadRequestError):
-            room_api.hls_subscribe(room.id, ["track-id"])
+            room_api.hls_subscribe(room.id, ["component-id"])
```

Comment:
What do you think about adding a test for a situation when we try to subscribe to a peer/component that doesn't exist?